fsharp-backend/src/LibBackend/Authorization.fs

```diff
@@ -152,7 +152,13 @@ let specialCasePermission
   (username : UserName.T)
   (ownerName : OwnerName.T)
   : Option<Permission> =
-  if List.any ((=) (ownerName, username)) specialCases then
+  if List.any
+       ((=)
+         (
+           ownerName,
+           username
+         ))
+       specialCases then
```

Comment:
I'm not experienced with Fantomas, but this seems much harder to read to me - are you happy with this formatting? (if so, ignore me, I suppose :))

Reply:
No, I hate this formatting and I'd love to figure out the switch to not do this.

Reply:
I love the idea of Fantomas, but was never able to get around this particular formatting awkwardness. I ended up not using Fantomas for my project for this reason. May be worth reaching out to Fantomas repo? I haven't seen a particular setting for this (checked after my above post) :(

Reply:
https://github.com/fsprojects/fantomas/issues/1700